analyzers/src/SonarAnalyzer.CSharp/Facade/CSharpSyntaxFacade.cs

```diff
@@ -70,17 +76,19 @@ node switch
             };
 
         public override SyntaxToken? NodeIdentifier(SyntaxNode node) =>
-            node switch
+            RemoveParentheses(node) switch
             {
                 AttributeArgumentSyntax attribute => attribute.NameColon?.Name.Identifier,
                 BaseTypeDeclarationSyntax baseType => baseType.Identifier,
+                ConditionalAccessExpressionSyntax conditionalAccess => NodeIdentifier(conditionalAccess.WhenNotNull),
```

Comment:
Let's remove this one. It's not used and it's not clear whether it should return identifier from `Expression` or `WhenNotNull` part from the `NodeIdentifier` method name.
```suggestion
```

Reply:
Fine with me. I'm pretty sure I added it, because I needed it, but that might be part of the fixer. We'll run into that later.